src/composables/useAppStore.ts

```diff
@@ -0,0 +1,37 @@
+import LS_KEYS from '@/constants/local-storage.keys';
```

Comment:
We already have a composable called `useApp` which was meant to be for this type of app-level state. Please just put this logic in there. Using the `Store` affix is a bit restrictive.